Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -43,7 +43,7 @@ public void SFTestSetup()
             log4net.Config.XmlConfigurator.Configure(logRepository, new FileInfo("App.config"));
 #endif
 
-            var reader = new StreamReader("parameters.json");
+            var reader = new StreamReader("C:\\Snowflake\\snowflake-connector-net\\Snowflake.Data.Tests\\parameters.json");

```

Comment:
Hardcoded path.

Reply:
Agree. We should not hard-code the path here. Actually, in `.csproj` file, I had a section that copy the parameters.json file into target directory so that you only need to specify `parameters.json`

Reply:
Thanks for pointing out. I added this full path and forgot to remove it. The reason back then was because I remove the net46 target in the .csproj and it stoped to copy the json.
Will revert this change